pipelines/smartseq2_single_sample/SS2SingleSample.wdl

```diff
@@ -0,0 +1,136 @@
+## ** New Line between task definitions
+## ** New lines between blocks in tasks
+## ** New line variable declarations that you want to group
+## ** New line between imports and workflow definition
+## ** No new lines at the start or end of any workflow/call/task definition block
+## ** No need for having two new lines anywhere
+## ** No new lines at the start of any blocks
+
+##### Spacing Rules #####
+## ** Space before `{` characters
+## ** Space after `,` characters
+## ** Space before and after operators e.g. `+`, `=`
+## ** Space after `:` character in runtime block
+## ** Space after `#` when creating a comment (or `##` if we decide thats our comment definer)
+
+##### Tab vs Space for Indents ######
+## ** Always use spaces? I think whats most frequently used is whenever you want to indent you use 2 spaces but we should decide what works best for us.
+
+##### Runtime Attributes ######
+## ** This will change quite a bit i believe when the new spec of WDL comes out (WDL SPEC 3) but we should define all values we care about
+## ** even if they are the default values.  e.g. write `cpu: 1` instead of leaving it out because its the default.
+
+##### Comment Rules ######
+## **  Should comments have one or two `##` before a comment.  Maybe two for workflow or task descriptions and one for any other kind of comment?
+
+import "Hisat2QcPipeline.wdl" as RunHisatQCWDL  ## ** Should import alias follow some kind of pattern like ending in "WDL" or something else
+import "Hisat2RsemPipeline.wdl" as RunHisatRSEMWDL
+
+## ** Explanations above the workflow about what inputs it expects, what outputs/metrics it outputs
+## ** What, if any, assumptions are made in the pipeline
+## ** In general any kind of description you want to add above the workflow is nice
+## ** A good example of this can be found in these wdls
+## ** https://github.com/broadinstitute/dsde-pipelines/blob/develop/genomes_in_the_cloud/single_sample/PairedSingleSampleWf.wdl
+## ** https://github.com/broadinstitute/gatk/blob/master/scripts/mutect2_wdl/mutect2.wdl
+
+## ** Should comments / descriptions above a task definition be mandatory?
+
+## main pipeline:
+## QC track: HISAT2+Picard
+## this track will produce aligned bam file and a set of QC metrics
+## rsem quantification track: HISAT2+RSEM
+## this track involves hisat2 transcriptome alignment and RSEM gene expression estimation.
+workflow SmartSeq2SingleCell {
+  # load annotation
+  File gtf_file
+  File genome_ref_fasta
+  File rrna_intervals
+  File gene_ref_flat
+  # load index
+  File hisat2_ref_index
+  File hisat2_ref_trans_index
+  File rsem_ref_index
+  # ref index name
+  File hisat2_ref_name
+  File hisat2_ref_trans_name
+  # samples
+  String stranded
+  String sample_name
+  String output_name
+  File fastq1
+  File fastq2
+
+  call RunHisatQCWDL.RunHisat2Pipeline as QC {  ## ** Should callign a subworkflow be aliased some suffix or prefix so its easier to know when we're dealing with a workflow as opposed to a task?
```

Comment:
100% yes. 

If you're calling a workflow, call it as `QC_workflow` -- makes it much, much clearer.